content/en/docs/instrumentation/php/automatic.md

```diff
@@ -57,6 +57,14 @@ function ends it.
       how to do that using a specific version of the extension (1.0.0beta2 in
       this case):
 
+      **Installing from source requires proper development environment and few
+      dependendencies:**
+
+      {{< ot-tabs "Linux (apt)" "MacOS (homebrew)">}}
```

Comment:
```suggestion
      {{< ot-tabs "Linux (apt)" "macOS (homebrew)">}}
```

Reply:
@chalin ok. I made changes based on https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/instrumentation/cpp/getting-started.md. Seems that it should be updated as well to be consistent.